packages/fiber/src/core/events.ts

```diff
@@ -61,20 +56,24 @@ export type EventHandlers = {
 }
 
 export type FilterFunction = (items: THREE.Intersection[], state: RootState) => THREE.Intersection[]
-export type ComputeOffsetsFunction = (
-  event: any,
-  state: RootState,
-) => { offsetX: number; offsetY: number; width?: number; height?: number }
+export type ComputeFunction = (event: DomEvent, root: RootState, previous?: RootState) => void
 
 export interface EventManager<TTarget> {
+  /** Determines if the event layer is active */
   enabled: boolean
+  /** Event layer priority, higher prioritized layers come first and may stop(-propagate) lower layer  */
   priority: number
+  /** The compute function needs to set up the raycaster and an xy- pointer  */
+  compute?: ComputeFunction
```

Comment:
Why not call this `setup`, `init` or `bootstrap`? Compute is super generic, might as well be `function` 😆

Reply:
its being called every frame though. i thought setup sounded too much like one-time